README.md

```diff
@@ -89,10 +89,21 @@ terraform {
 }
 ```
 
+### Installing pre-commit hooks
+For consistent coding style, it's recommended to install [pre-commit](https://pre-commit.com/#install) hooks.
```

Comment:
@avnerenv0 
Without installing the pre-commit hooks on your local machine, nothing is enforced.
It's possible you can add it to the pipeline, and fail it in the pipeline. (But that's about it).

Reply:
so IMO if we're not enforcing this we'll end up with this PR again in a couple of months, can we add it as a check to the CI ?

Reply:
at least as a requirement to merge PRs

Reply:
@avnerenv0 - running pre-commit hooks in the pipeline is not advisable (because it's for git commits).
Fortunately, I added more checks to the pipeline. See: 605e7a3b6cf6c369c8fae3eb08f37a210a5e9a4e

It should cover most of the use-cases.

Reply:
sorry - this one:
480d6c0d80c7b4c04e1e3489a8706cb46ea15cc7